src/cli/cli_mac_filter.cpp

```diff
@@ -330,6 +391,24 @@ template <> otError MacFilter::Process<Cmd("rss")>(Arg aArgs[])
             error = otLinkFilterAddRssIn(GetInstancePtr(), &extAddr, rss);
         }
     }
+    /**
+     * @cli macfilter rss remove
+     * @code
+     * macfilter rss remove *
+     * Done
+     * @endcode
+     * @code
+     * macfilter rss remove 0f6127e33af6b404
+     * Done
+     * @endcode
+     * @cparam macfilter rss remove @ca{extaddr}
+     * @par api_copy
+     * #otLinkFilterRemoveRssIn
+     * @par
+     * If you wish to remove the default received signal strength and link quality indicator settings,
```

Comment:
This is similar guidance to other commands, which we moved after the cparam tag... should we move this one too?